demo/src/docs/ColorPalette.js

```diff
@@ -0,0 +1,256 @@
+                                color: Color(
+                                  theme.colors[color][subColor]
+                                ).isDark()
+                                  ? theme.colors.white
+                                  : theme.colors.gray800,
+                              }}
+                            >
+                              {subColor}
+                            </div>
+                          </div>
+                        ))}
+                      </div>
+                    </Fragment>
+                  );
+                }
+              })}
+            </Fragment>
+          )}
+
+        {/* Only bringing back black & white
+      since those are the only other colors that need to be displayed
+    and are also not part of an object */}
+        {theme &&
+          theme.colors && (
+            <div>
+              <Typography type="heading5" style={{ margin: "24px 0px" }}>
+                Other Colors
+              </Typography>
+              <div className={classes.bottomContainer}>
+                {Object.keys(theme.colors).map((color) => {
+                  if (
+                    (!isObject(theme.colors[color]) &&
+                      theme.colors[color] === theme.colors.black) ||
+                    theme.colors[color] === theme.colors.white
+                  ) {
+                    return (
+                      <div
+                        key={color}
+                        style={{ background: theme.colors[color] }}
+                        className={classNames(
+                          classes.otherColor,
+                          theme.colors[color] === theme.colors.white &&
+                            classes.addBorder
+                        )}
+                      >
+                        <div
+                          className={classes.subColorText}
+                          style={{
+                            color: Color(theme.colors[color]).isDark()
+                              ? theme.colors.white
+                              : theme.colors.gray800,
+                          }}
+                        >
+                          {color}
+                        </div>
+                        <div
+                          className={classes.subColorText}
+                          style={{
+                            color: Color(theme.colors[color]).isDark()
+                              ? theme.colors.white
+                              : theme.colors.gray800,
+                          }}
+                        >
+                          {theme.colors[color]}
+                        </div>
+                      </div>
+                    );
+                  }
+                })}
+              </div>
+            </div>
+          )}
+      </div>
+    </div>
+  );
+};
+
+const styles = (theme) => ({
+  wrapper: {
+    maxWidth: "680px",
+    width: "100%",
+    margin: "45px auto",
```

Comment:
Take a look at most of the other Component docs. There are only a few, like this one, that are centered in the right container of the screen. The rest are left aligned. Personally, I prefer the left aligned ones to centered, but ultimately its your choice.